bundle/jsky.elevation.plot/src/main/java/jsky/plot/ElevationPanel.java

```diff
@@ -127,33 +126,30 @@
     // Options for secondary Y axis
     public static final String Y2_AXIS_AIRMASS = _I18N.getString("Airmass");
     public static final String Y2_AXIS_PA = _I18N.getString("ParallacticAngle");
-//    public static final String[] Y2_AXIS_OPTIONS = {
-//        Y2_AXIS_AIRMASS, Y2_AXIS_PA
-//    };
     private String _y2AxisLabel = Y2_AXIS_AIRMASS;
 
-    // Used to display the degrees values in the prinary axis in the range 0..90
-    private static NumberFormat _yAxisNumberFormat = new DecimalFormat() {
-            public StringBuffer format(double number, StringBuffer result,
-                                       FieldPosition fieldPosition) {
-                if (number > 90.) {
-                    return result; // used to leave blank space for labels above 90 deg mark
-                }
-                return super.format(number, result, fieldPosition);
+    // Used to display the degrees values in the primary axis in the range 0..90
+    private static final NumberFormat _yAxisNumberFormat = new DecimalFormat() {
+        @Override
+        public StringBuffer format(double number, StringBuffer result, FieldPosition fieldPosition) {
+            if (number > 90.) {
```

Comment:
`90.0` please, here and elsewhere.

Reply:
Last time I touched additional code, given proximity to release, I was told to make the minimal changes. All of this is existing code. I'm more than happy to check it and change it, provided my efforts will not be wasted.

Reply:
That's a reasonable point. Maybe just change it in your code. Allan used to do it all the time and we need to get out of the habit.